stac-example/src/main/scala/geotrellis/stac/raster/StacRepository.scala

```diff
@@ -1,51 +1,34 @@
+/*
```

Comment:
Looks like this implementation is only used in tests. Can it be moved to the `test` package?